css/style.css

```diff
@@ -815,6 +815,10 @@ HEAD .round-2 {
   .box {
     margin-bottom: 20px;
   }
+  .container
+  {
+    flex-direction: column;
```

Comment:
Why changing the flex-direction?

Reply:
Sir I thought it will make it responsive like in mobile size it will come to center. All other issues i have resolved today. Only this responsive issue is left to be resolved which I will complete soon...